src/emulator/apphosting/serve.ts

```diff
@@ -7,8 +7,14 @@ import { isIPv4 } from "net";
 import { checkListenable } from "../portUtils";
 import { discoverPackageManager } from "./utils";
 import { DEFAULT_HOST, DEFAULT_PORTS } from "../constants";
-import { wrapSpawn } from "../../init/spawn";
+import { spawnWithCommandString, wrapSpawn } from "../../init/spawn";
 import { getLocalAppHostingConfiguration } from "./config";
+import { logger } from "./utils";
+import { Emulators } from "../types";
+
+interface StartOptions {
+  customStartCommand?: string;
```

Comment:
optional naming nit: `startCommand`? marking it optional gets the point across imo but up to you